packages/uui-button/lib/uui-button.story.ts

```diff
@@ -11,13 +11,21 @@ export default {
   id: 'uui-button',
 
   args: {
+    href: undefined,
+    target: undefined,
     look: 'default',
     color: 'default',
     type: '',
     label: 'Button',
     state: '',
   },
   argTypes: {
+    href: {
+      control: { type: 'text' },
```

Comment:
Doesn't it do this out of the box without explicitly setting it to text?

Reply:
Seems like a brain fart.